### ISSUE TITLE
Rd warnings when installing

### ISSUE THREAD

**Issue by bdsimpson** [elbersb/tidylog#11]
I like this package so far.
I am receiving a warning during installation regarding file links to 'inner_join' and 'transmute' in dplyr.

This seems related to an issue opened in roxygen:
https://github.com/klutometis/roxygen/issues/707

My setup:

- R version 3.5.2
- Windows 10
- Installing with packrat on

Install messages:
```
* installing *source* package 'tidylog' ...
** R
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
  converting help for package 'tidylog'
    finding HTML links ... done
    filter                                  html  
    group_by                                html  
    inner_join                              html  
Rd warning: C:/Users/Byron/AppData/Local/Temp/Rtmp4K2WMi/R.INSTALL4a2837776241/tidylog/man/inner_join.Rd:26: file link 'inner_join' in package 'dplyr' does not exist and so has been treated as a topic
Rd warning: C:/Users/Byron/AppData/Local/Temp/Rtmp4K2WMi/R.INSTALL4a2837776241/tidylog/man/inner_join.Rd:28: file link 'inner_join' in package 'dplyr' does not exist and so has been treated as a topic
Rd warning: C:/Users/Byron/AppData/Local/Temp/Rtmp4K2WMi/R.INSTALL4a2837776241/tidylog/man/inner_join.Rd:31: file link 'inner_join' in package 'dplyr' does not exist and so has been treated as a topic
    mutate                                  html  
    select                                  html  
    tidylog                                 html  
    transmute                               html  
Rd warning: C:/Users/Byron/AppData/Local/Temp/Rtmp4K2WMi/R.INSTALL4a2837776241/tidylog/man/transmute.Rd:20: file link 'transmute' in package 'dplyr' does not exist and so has been treated as a topic
Rd warning: C:/Users/Byron/AppData/Local/Temp/Rtmp4K2WMi/R.INSTALL4a2837776241/tidylog/man/transmute.Rd:22: file link 'transmute' in package 'dplyr' does not exist and so has been treated as a topic
Rd warning: C:/Users/Byron/AppData/Local/Temp/Rtmp4K2WMi/R.INSTALL4a2837776241/tidylog/man/transmute.Rd:25: file link 'transmute' in package 'dplyr' does not exist and so has been treated as a topic
** building package indices
** testing if installed package can be loaded
*** arch - i386
*** arch - x64
* DONE (tidylog)
In R CMD INSTALL
```

**Comment by bdsimpson** [comment]
I opened a pull request to fix the links

**Comment by elbersb** [comment]
fixed by #14
